### ISSUE TITLE
windows build

### ISSUE THREAD

**Issue by artofmus** [logsol/JuceLibPd#2]
I'm not really friends with the low-level programming. On this issue, you plan to release here builds for windows? That would be super!

**Comment by logsol** [comment]
Yes we are indeed! We are currently in the process of setting up a windows deployment. I will close this issue once its published.

**Comment by eduardopatricio** [comment]
Brilliant! Thank you!!

**Comment by alfonso73** [comment]
hi guys! great! something goin' on on the Win 32/64 bit port? bests!

**Comment by logsol** [comment]
sorry, not yet but any help with it is much appreciated!

**Comment by alfonso73** [comment]
I'll try to compile on Windows.
Is there any specific OSX code that i have to know about? ( i mean specific
c++ libs etc.)

www.elgallorojorecords.com
soundcloud.com/alfonsosantimone
www.facebook.com/alfonsosantimone

**Comment by logsol** [comment]
It's all standard JUCE stuff. The Pd Pulp code should compile out of the box. The only dependency is libpd and what we didn't get to work was compiling libpd using MinGW as described here https://github.com/libpd/libpd

**Comment by logsol** [comment]
this might be a starting point http://www.thefuntastic.com/2014/04/the-misadventures-of-unity-and-puredata-libpd/

**Comment by artofmus** [comment]
Any success?
